vinay1.c

```diff
@@ -0,0 +1,93 @@
+#include<stdio.h>
```

Comment:
Please move your implementation file to the correct folder. Check [CONTRIBUTING.md](../../blob/master/CONTRIBUTING.md)